XenServerHealthCheck/XenServerHealthCheckBugTool.cs

```diff
@@ -156,34 +156,34 @@ public void RunBugtool(IXenConnection connection, Session session)
 
             string timestring = DateTime.Now.ToString("yyyy-MM-dd-HH-mm-ss");
 
-            // Collect all master/slave information to output as a separate text file with the report
-            List<string> mastersInfo = new List<string>();
+            // Collect all coordinator/supporter information to output as a separate text file with the report
+            List<string> coordinatorInfo = new List<string>();
 
             int i = 0;
             Pool p = Helpers.GetPool(connection);
             foreach (Host host in connection.Cache.Hosts)
             {
-                // master/slave information
+                // coordinator/supporter information
                 if (p == null)
                 {
-                    mastersInfo.Add(string.Format("Server '{0}' is a stand alone server",
+                    coordinatorInfo.Add(string.Format("Server '{0}' is a stand alone server",
                         host.Name()));
                 }
                 else
                 {
-                    mastersInfo.Add(string.Format("Server '{0}' is a {1} of pool '{2}'",
+                    coordinatorInfo.Add(string.Format("Server '{0}' is a {1} of pool '{2}'",
                         host.Name(),
-                        p.master.opaque_ref == host.opaque_ref ? "master" : "slave",
+                        p.master.opaque_ref == host.opaque_ref ? "master" : "supporter",
```

Comment:
2 masters

Reply:
Since we're dealing with variables connected to XenAPI, I didn't want to add breaking changes. Again @kc284 maybe you can advise.

Reply:
if its api then lets not break it.

Reply:
This is not API, but it could be in use by the scripts analysing the status reports uploaded, so best keep it as is.

Reply:
Hmm, on second thought, this is only used to format the string, so I doubt the scripts parse the string. Nah, I think I was wrong the first time, we can change this.